package.json

```
@@ -8,6 +8,14 @@
     "tap": "bin/run.js"
   },
   "main": "lib/tap.js",
+  "exports": {
+    ".": {
+      "import": "./lib/tap.mjs",
+      "default": "./lib/tap.js"
+    },
```

```suggestion
    ".": [
      {
        "import": "./lib/tap.mjs",
        "default": "./lib/tap.js"
      },
      "./lib/tap.js"
    ],
```
this is required for node v13.0 and v13.1.

---

Please don't encourage this pattern.

---

I will continue to do so, and will be using it on all 300 of my packages as i gradually updated them to have “exports”, since we chose not to backport patches to those minors.

Since this package lacks an engines field, these node versions are supported, so not accepting this change makes this semver-major, even with the slash pattern fallback.

---

Node 13 is already past end-of-life.

---

Which means node doesn’t support it anymore, but that doesn’t free packages from semver. Most of my packages support down to node 0.4, and will continue to do so for the foreseeable future.

---

@ljharb I don't agree with that assessment. While an engines field can be used to infer a support matrix I think it is a bit of a stretch to imply a lack of one implies support for all version. AFAIK there is no easy way in an engines field to say "this supports 10, 12, 14, 15, but not 11 and 13". 

Also fwiw the package has an engines field that says ">=8"

---

I also do not feel this is something which needs to be addressed.  Node.js 13.0 and 13.1 are broken, node.js 13.2.0+ is not.  If this broke all of 13.x I'd feel differently but the release family still works (just not the broken minor releases).

---

@mylesborins That’s all fine; you asked me to open this suggestion so you’d see what it would look like.

Thanks, I’d missed the engines field; the easiest way is `^10 || ^12 || ^14 || ^15`, ftr.